ajax/libs/emojione/package.json

```
@@ -1,10 +1,10 @@
 {
   "name": "emojione",
   "title": "emojione",
-  "filename": "lib/js/emojione.min.js",
-  "version": "2.2.7",
+  "filename": "js/emojione.min.js",
+  "version": "3.1.6",
```

```suggestion
  "version": "2.2.7",
```

Revert version change, auto-update bot will handle this.